eppoclient/assignmentlogger.go

```diff
@@ -7,13 +7,13 @@ type IAssignmentLogger interface {
 }
 
 type AssignmentEvent struct {
-	Experiment        string
-	FeatureFlag       string
-	Allocation        string
-	Variation         Value
-	Subject           string
-	Timestamp         string
-	SubjectAttributes dictionary
+	Experiment        string     `json:"experiment"`
```

Comment:
Heads up, seeing tab characters in here (vs. spaces)

[Relevant Silicon Valley clip](https://www.youtube.com/watch?v=SsoOG6ZeyUI).